README.md

```diff
@@ -289,6 +289,58 @@ export GITHUB_TOKEN=$(gh auth token)
 binst check
 ```
 
+## 📋 Schema Command
```

Comment:
I think it's better to show this section after or inside Configuration format section.